51DA/MANIFEST

```diff
@@ -0,0 +1,3 @@
+purpose=xyz.openbmc_project.Software.Version.VersionPurpose.PSU
+version=393141313900
```

Comment:
how did you determine this version number?  looks like this corresponds to "19.1A.09", but I don't think the 00 is valid.  Was this at offset 0x10 in the image?